webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/TableIndexPage.js

```diff
@@ -58,8 +58,9 @@ A page component that displays a table with data fetched from the API. It provid
 @param {Array<Object>} {customToolbarItems} - an array of custom toolbar items to be displayed
 @param {boolean} {exportable} - whether or not to show export button
 @param {boolean} {hasHelpPage} - whether or not to show documentation button
-@param {string}{headerText} - the header text for the page
-@param {string}{header} - header node; default is <title>{headerText}</title>
+@param {React.ReactNode}{customHeader} - a custom header to be rendered instead of the default header
+@param {string}{headerText} - DEPRECATED - the header text for the page
```

Comment:
Can't we delete the `headerText`?

Reply:
no since its used in Katello: https://github.com/Katello/katello/blob/master/webpack/components/extensions/Hosts/BulkActions/HostReview.js#L125